### PR TITLE
[HSEARCH] 5.8.0.Beta2 release

### PR DESCRIPTION
**Pull request by yrodiere**
See the post on http://staging.in.relation.to/2017/05/11/hibernate-search-5-8-0-Beta2/

### CODE REVIEW

File: posts/Yoann/2017-05-11-hibernate-search-5-8-0-Beta2.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+====
+Hibernate Search 5.8.0.Beta2, just as 5.7.0.Final, is only compatible with Hibernate ORM 5.2.3 and later.
+
+If you need to use Hibernate ORM 5.0.x or 5.1.x and cannot upgrade,
+please use Hibernate Search 5.6.1.Final.
+====
+
+== What's new since the first Beta?
+
+For a full list of changes since 5.8.0.Beta1, please see https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=29000[the release notes].
```
Comment (gsmet) [inline posts/Yoann/2017-05-11-hibernate-search-5-8-0-Beta2.adoc]: If you want to put backticks around version numbers, let's be consistent!

File: posts/Yoann/2017-05-11-hibernate-search-5-8-0-Beta2.adoc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Below is a summary of the main changes:
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2606[HSEARCH-2606]: duplicate parameters in analyzer definitions
+   are now detected automatically and trigger an error on startup.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2014[HSEARCH-2014] (local Lucene indexes only):
+   index size is now reported as part of Hibernate Search statistics,
+   available over JMX in particular if enabled.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2208[HSEARCH-2208]: use of `org.apache.lucene.search.Filter`
+   throughout the APIs has been deprecated. You should use `org.apache.lucene.search.Query` instead,
+   because `org.apache.lucene.search.Filter` will be removed in the next version of Lucene.
```
Comment (gsmet) [inline posts/Yoann/2017-05-11-hibernate-search-5-8-0-Beta2.adoc]: Well, tbh, it has already been removed. Not sure how to phrase it though.

**Review by gsmet — COMMENTED** [review]
Looks good. Added a couple of minor comments inline. Feel free to ignore them :).

One thing I think might be good to add is quick summary of what's in 5.8. Maybe an "About 5.8" paragraph at the start. WDYT?

**Comment by yrodiere** [comment]
@gsmet Thanks, I followed your advice and force pushed. I will have a look at how it's rendered on staging and will merge.

**Pull request merged**